ansible/deploy_regular_node.yml

```diff
@@ -1,40 +1,98 @@
 ---
 
 - hosts: all
-  remote_user: ubuntu
+  remote_user: '{{ remote_user }}'
+  environment:
+    PATH: /usr/local/go/bin:{{ ansible_env.PATH }}
 
   tasks:
-    - name: Install docker
+    - name: Add the deploy user
       become: yes
-      apt:
-        name: docker.io
+      user:
+        name: '{{ deploy_user }}'
         state: present
+    
+    - name: Install dependencies
```

Comment:
I'd separate basic requirements (as the apt packages) form the more specific quorum installation, just to make it easy to deploy separately using tags. We might want to install a new version of quorum but not launch the apt for example.

Reply:
I think that is how the deploy works now. The apt dependencies and the quorum installation are separated or i'm misunderstanding something?